src/frontend/get-github-setup.ts

```diff
@@ -14,17 +14,39 @@ export default async (req: Request, res: Response): Promise<void> => {
 	req.log.info("Received get github setup page request");
```

Comment:
I wonder if we should update the method readme? It says i handles redirects from Jira, however it doesn't seem so (according to videos)

Reply:
Good catch.

Reply:
Just looked. It says it handles redirects specifically for the installation flow for Jira and GH. However, the rest of the comments are outdated so have amended those.